awx/ui_next/src/components/Lookup/Lookup.jsx

```diff
@@ -177,11 +201,11 @@ class Lookup extends React.Component {
   saveModal() {
     const { onLookupSave, name, multiple } = this.props;
     const { lookupSelectedItems } = this.state;
-    const value = multiple
-      ? lookupSelectedItems
-      : lookupSelectedItems[0] || null;
-    onLookupSave(value, name);
-    this.handleModalToggle();
+    const value = multiple ? lookupSelectedItems : lookupSelectedItems[0] || null;
```

Comment:
I plan to offload all of the state management back onto `Lookups` parents, but this work.